pkg/api/controller.go

```diff
@@ -56,6 +56,10 @@ const (
 	setupStateNotInitialized = "not_initialized"
 
 	DefaultMaxDeleteObjects = 1000
+
+	ResetPasswordAudience = "password_reset"
+	tokenExpiryDuration   = 5 * time.Minute
+	loginAudiance         = ""
```

Comment:
```suggestion
	loginAudience         = ""
```

Consistent spelling here is **critical**.  If I need to verify all audiences for a security audit, I will miss this audience.  Also, ideally spelling of this word would be correct in the comments below, too.